themes/bootstrap3/templates/myresearch/editlist.phtml

```diff
@@ -18,14 +18,14 @@
 <h2><?=$this->transEsc($pageTitle); ?></h2>
 
 <form class="form-edit-list" method="post" name="<?=$this->newList ? 'newList' : 'editListForm'?>">
-  <input type="hidden" name="id" value="<?=empty($this->list->id) ? 'NEW' : $this->escapeHtmlAttr($this->list->id) ?>">
+  <input type="hidden" name="id" value="<?=empty($this->list->getId()) ? 'NEW' : $this->escapeHtmlAttr($this->list->getId()) ?>">
   <div class="form-group">
     <label class="control-label" for="list_title"><?=$this->transEsc('List'); ?>:</label>
-    <input id="list_title" class="form-control" type="text" name="title" value="<?=isset($this->list['title']) ? $this->escapeHtml($this->list['title']) : ''?>">
+    <input id="list_title" class="form-control" type="text" name="title" value="<?=$this->list->getTitle() ?? ''?>">
```

Comment:
If you change the return value of getTitle from `?string` to `string` as suggested above, then you don't need the `?? ''` here since it will be handled inside the getter method.